### ISSUE TITLE
Midterm exam

### ISSUE THREAD

**Issue by jeehoonkang** [snu-sf-class/pl2016#56]
Midterm exam will be held.
- Time: Apr 16 (Sat) 18:00-21:00
- Place: Bldg 302 Rm 312 Software Lab

**Comment by taehwoi** [comment]
시험 때는 coqide를 사용해야 하는지 아니면 아무 에디터라도 괜찮을지 궁금합니다.
소프트웨어 실습실에서 개인 계정을 사용하면 상관 없는데 PL전용 계정을 사용해야 한다면 미리 개발환경을 셋팅 해놓을 수 있을까요?
감사합니다.

**Comment by jeehoonkang** [comment]
@indiofish 
- ~~개인 계정을 사용해도 좋습니다.~~
- ~~CoqIDE가 기본으로 제공되지만 필요에 따라 다른 IDE를 사용하셔도 좋습니다.~~

**Comment by jeehoonkang** [comment]
답변을 보류하겠습니다. 조만간 다시 답변드리겠습니다.

**Comment by alxest** [comment]
저도 미리 제 개발환경을 셋팅해놓거나, 제 노트북에서 할 수 있으면 좋겠습니다.

**Comment by jeehoonkang** [comment]
Please reply here if you want to use your own laptop for the exam.

If you use Vim or Emacs (or other editors) for Coq development, please say so.  Those who use other editors than CoqIDE will have the priority for using laptops.

**Comment by ghost** [comment]
I use Vim and I would like to use my laptop for the midterm. 
Thank you.

**Comment by swchoi06** [comment]
I use Vim also.

Thank you.

**Comment by taehwoi** [comment]
I use vim and I would also like to use my laptop.
Thank you.

**Comment by hestia49** [comment]
저도 빔을 사용합니다. coq IDE는 한번도 사용해본적 없습니다. 랩탑을 사용하기를 원합니다.
감사합니다.

**Comment by alxest** [comment]
emacs를 사용하고 dvorak을 씁니다. 노트북을 사용하기를 원합니다.

**Comment by nolsigan** [comment]
I use Vim also

Thank you

**Comment by HanInSeoung** [comment]
랩탑을 사용하고 싶습니다.

**Comment by shwlinux** [comment]
노트북을 사용하고 싶습니다.

**Comment by msgundam00** [comment]
vim을 사용하고 있고 개인 노트북 사용하고 싶습니다.

**Comment by isutare412** [comment]
I also use vim in laptop.
Thank you.

**Comment by nimooj** [comment]
저도 vim을 사용해서 노트북 사용하고 싶습니다. 감사합니다.

**Comment by jeehoonkang** [comment]
공정한 추첨에 의해 (`cat ids.txt | sort -R`) 다음 분들이 노트북을 사용하실 분들로 선정되었습니다:

@sophia5787
@nolsigan
@nimooj
@alxest
@indiofish

**Comment by jeehoonkang** [comment]
2015년 중간고사 업로드했습니다: https://github.com/snu-sf-class/pl2016/blob/master/exams/2015mid.v

**Comment by k32l** [comment]
Was the syllabus for pl2015 different? In the provided example of midterm there is a list of tactics and tacticals. However there are at least 6 or 7 tactics that we didn't learned yet until now.

**Comment by k32l** [comment]
I was wondering if it is possible to bring blank sheet to make notes?

**Comment by jeehoonkang** [comment]
@k32l Yeah, last year students learned more tactics as of the midterm.  You can take your blank sheets.

**Comment by jeehoonkang** [comment]
We decided to allow all Vim/Emacs users to use your own laptop.  Bring your laptop if you want to use it in the exam.

**Comment by jeehoonkang** [comment]
The midterm score is announced: http://147.46.15.109:9480/exam

**Comment by jeehoonkang** [comment]
이의신청은 이번주 금요일까지 받겠습니다 (이메일/issue tracker로).  그 이후에 제기하신 이의는 반영하지 않겠습니다.

**Comment by jeehoonkang** [comment]
지금까지 접수된 이의:
- https://github.com/snu-sf-class/pl2016/issues/66
- https://github.com/snu-sf-class/pl2016/issues/64

**Comment by jeehoonkang** [comment]
오늘까지 접수된 claim을 반영하였습니다: http://147.46.15.109:9480/exam

**Comment by nolsigan** [comment]
페이지에 접속이 안됩니다

500 error
